README.md

```diff
@@ -1,4 +1,4 @@
-  <h3>@Mumbereh</b></h3>
+  <h3>Readme Templete for Blog App/b></h3>
```

Comment:
[OPTIONAL] I think you should remove this. People don't need to know that you are using a template and your `<b/>` tag is showing